docs/learn/learn-comparisons.md

```diff
@@ -1,63 +1,196 @@
-
-Polkadot has an entirely different purpose, as it was built to connect and secure unique
-blockchains. It is a protocol on which single blockchains (such as Binance Smart Chain) could be
-built and benefit from shared security, interoperability and scalability. Interoperability within
-Polkadot is based on pooled security on Polkadot, and the security of the entire Polkadot network,
-and has much stronger economic security.
-
-Scalability based on bridges relies on each bridged chain finding its own set of validators,
-therefore duplicate resources are required. Scalability on Polkadot is based on the security of the
-relay chain, and as the number of validators in the active set on Polkadot are increased, more
-parachains can be supported.
+In this section we explore the main differences in rollup technology between:
+
+- Zero-knowledge rollups,
+- Optimistic rollups, and
+- Polkadot rollups (i.e. parachains).
+
+Before diving into the differences between these rollup technologies, it is important to understand
+the difference between non-interactive and interactive methods.
+
+### Non-interactive vs. Interactive Rollup Methods
+
+In the context of rollups, **interactive** and **non-interactive** methods refer to how the validity
+of transactions or state changes is established between the rollup layer and the main blockchain
+(L1).
+
+A non-interactive method relies on cryptographic proofs to validate transactions or state changes
+without requiring dispute resolution or interactive challenges. A prominent example is
+zero-knowledge proving mechanisms that operate off-chain, such as a zk-SNARK or zk-STARK. These
+protocols prove that the transactions in a batch satisfy the rules of the protocol. These proofs are
+submitted to the L1 chain and verified without needing further interaction.
+
+An interactive method involves a back-and-forth process between parties to validate transactions. A
+prominent example is the usage of fraud proofs, as used in optimistic rollups. If a dispute arises,
+a challenge mechanism is triggered, where a "prover" submits evidence (fraud proof) to demonstrate
+that a transaction is invalid. This can require multiple steps, with both the challenger and
+defender submitting data to the L1 chain.
+
+Here below, the key differences between interactive and non-interactive methods.
+
+| **Aspect**         | **Interactive**                                 | **Non-Interactive**                                                     |
+| ------------------ | ----------------------------------------------- | ----------------------------------------------------------------------- |
+| **Mechanism**      | Back-and-forth dispute resolution.              | One-time proof submission and verification.                             |
+| **Latency**        | Long finality due to challenge periods.         | Near-instant finality.                                                  |
```

Comment:
So this isn't really true. In practice, Polkadot has faster finality and zk rollups take hours to finalize. zksync for instance takes a whole day.